bw2data/backends/proxies.py

```diff
@@ -71,7 +72,7 @@ def __init__(self, key, kinds=None, reverse=False):
     def filter(self, expr):
         self._args.append(expr)
 
-    def delete(self):
+    def delete(self, signal: bool = True):
```

Comment:
Is the parameter intentionally ignored (I'm not sure Python has a syntax to explicitly state that)?